src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -249,9 +266,18 @@ private void OnPreRender()
         {
             Graphics.Blit(CurrentColorTexture, colorRGBTexture, CurrentColorMaterial);
 
-            // Clear the camera's background by blitting using a shader that simply
-            // clears the target texture without reading from the source texture.
-            Graphics.Blit(null, spectatorViewCamera.targetTexture, textureClearMat);
+            if (IsVideoOutputQuadrantMode)
+            {
+                // Clear the camera's background by blitting using a shader that simply
+                // clears the target texture without reading from the source texture.
+                Graphics.Blit(null, spectatorViewCamera.targetTexture, textureClearMat);

```

Comment:
nit: maybe call out that this is only required for recording in quad compared to displaying in quad?